### PR TITLE
@JoshDreamland fuck joo bitch

### PR DESCRIPTION
**Pull request by time-killer-games**
fuck joo for merging a pr that didn't pass ci

### CODE REVIEW

**Comment by time-killer-games** [comment]
@RobertBColton well boo, it didn't do the job.

This might, however:
```
#ifndef PCHANGEFILTERSTRUCT
#define PCHANGEFILTERSTRUCT
#endif
```

**Comment by time-killer-games** [comment]
@RobertBColton lel thanks for the commits. That's exactly why I keep checked "allow edits from maintainers". This has proven to be a real pain. But I'm glad it appears you fixed it.

File: ENIGMAsystem/SHELL/Universal_System/Extensions/FileDropper/FileDropper.cpp [hunk 1]
```diff
@@ -96,9 +96,18 @@ LRESULT CALLBACK SetHook(int nCode, WPARAM wParam, LPARAM lParam) {
 }
 
 HHOOK InstallHook() {
+  // Windows Vista
+  #if (_WIN32_WINNT == _WIN32_WINNT_VISTA)
```
Comment (RobertBColton) [inline ENIGMAsystem/SHELL/Universal_System/Extensions/FileDropper/FileDropper.cpp]: Are you sure this is working locally though? I don't know if `_WIN32_WINNT` is actually being defined by anything.
https://docs.microsoft.com/en-us/cpp/porting/modifying-winver-and-win32-winnt?view=vs-2019

Comment (time-killer-games) [reply ENIGMAsystem/SHELL/Universal_System/Extensions/FileDropper/FileDropper.cpp]: It works with microsoft's compiler with my GM extension which effectively does the same thing, and I didn't anticipate MinGW being this big of a turd that it wouldn't support it.

**Pull request merged**